tests/simulator/refinement/harris.py

```
@@ -0,0 +1,225 @@
+    FluidDiagnostics,
+    ParticleDiagnostics,
+    InfoDiagnostics,
+)
+from pyphare.pharein import MetaDiagnostics
+from pyphare.pharein import ElectronModel
+from pyphare.simulator.simulator import Simulator, startMPI
+from pyphare.pharein import global_vars as gv
+from pyphare.pharein import LoadBalancer
+import sys
+import numpy as np
+import matplotlib.pyplot as plt
```

Ruff

`matplotlib.pyplot` imported but unused

Remove unused import: `matplotlib.pyplot`

(F401)